src/shared/DashBoardControls/DashBoardControls.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from "react";
+import { useNavigate } from "react-router-dom";
+
+import Box from "@mui/material/Box";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import Button from "@mui/material/Button";
+import AddIcon from "@mui/icons-material/Add";
+import { styledDashboard } from "./style";
+
+const DashBoardControls: FC = () => {
+  const navigate = useNavigate();
+
+  const addNewVacation = (): void => {
+    navigate("/user/vacation", { replace: true });
```

Comment:
Why do you use this prop for navigation?
{ replace: true }

Reply:
I was wrong about how it prop works. My bad)